primitives/primproc/primproc.cpp

```diff
@@ -147,7 +148,7 @@ void setupSignalHandlers()
 
 void setupCwd(Config* cf)
 {
-    string workdir = cf->getConfig("SystemConfig", "WorkingDir");
+    string workdir = startup::StartUp::tmpDir();
 
     if (workdir.length() == 0)
         workdir = ".";
```

Comment:
Not related to the diff.  Why does primproc need a particular working directory?